cirq-google/cirq_google/devices/grid_device_test.py

```diff
@@ -418,89 +418,44 @@ def test_to_proto_invalid_input(error_match, qubits, qubit_pairs, gateset, gate_
 
 
 def test_to_proto_backward_compatibility():
-    # Deprecations: cirq_google.SerializableGateSet and
-    # cirq_google.device.known_devices.create_device_proto_for_qubits()
-    with cirq.testing.assert_deprecated(
-        'SerializableGateSet',
-        'create_device_specification_proto()` can be used',
-        deadline='v0.16',
-        count=None,
-    ):
-        device_info, _ = _create_device_spec_with_horizontal_couplings()
-
-        # The set of gates in gate_durations are consistent with what's generated in
-        # _create_device_spec_with_horizontal_couplings()
-        base_duration = cirq.Duration(picos=1_000)
-        gate_durations = {
-            cirq.GateFamily(cirq_google.SYC): base_duration * 0,
-            cirq.GateFamily(cirq.SQRT_ISWAP): base_duration * 1,
-            cirq.GateFamily(cirq.SQRT_ISWAP_INV): base_duration * 2,
-            cirq.GateFamily(cirq.CZ): base_duration * 3,
-            cirq.GateFamily(cirq.ops.phased_x_z_gate.PhasedXZGate): base_duration * 4,
-            cirq.GateFamily(
-                cirq.ops.common_gates.ZPowGate, tags_to_ignore=[cirq_google.PhysicalZTag()]
-            ): base_duration
-            * 5,
-            cirq.GateFamily(
-                cirq.ops.common_gates.ZPowGate, tags_to_accept=[cirq_google.PhysicalZTag()]
-            ): base_duration
-            * 6,
-            cirq.GateFamily(cirq_google.experimental.ops.coupler_pulse.CouplerPulse): base_duration
-            * 7,
-            cirq.GateFamily(cirq.ops.measurement_gate.MeasurementGate): base_duration * 8,
-            cirq.GateFamily(cirq.ops.wait_gate.WaitGate): base_duration * 9,
-        }
-
-        # Serialize the old way
-        spec = known_devices.create_device_proto_for_qubits(
-            device_info.grid_qubits,
-            device_info.qubit_pairs,
-            [cirq_google.FSIM_GATESET],
-            known_devices._SYCAMORE_DURATIONS_PICOS,
-        )
+    device_info, _ = _create_device_spec_with_horizontal_couplings()
```

Comment:
This test can be removed, since we no longer need to be backward compatible with SerializableGateSet

Reply:
done